python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -156,3 +154,68 @@ def compile(model: nn.Module,
                 return pl_model
         else:
             return pl_model
+
+    def quantize(self, pl_model, calib_dataloader, val_dataloader=None, metric: Metric = None,
```

Comment:
Does `dynamic quantization` also need `calib_dataloader`?

Reply:
It's needed.

Reply:
both pytorch and onnx runtime do not need calibration dataset, why does INC need it? To compute the accuracy?
https://pytorch.org/tutorials/recipes/recipes/dynamic_quantization.html
https://onnxruntime.ai/docs/performance/quantization.html#quantization-api

Reply:
Correct. I fixed this in new push, `calib_dataloader` is optional now.